### PR TITLE
Updates ready for update 10 release

### PR DESCRIPTION
**Pull request by mbcoder**

### CODE REVIEW

**Review by JonLavi — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ plugins {
 }
```
Comment (JonLavi) [inline build.gradle:20]: The copyright at the top of the file is also on 2019

File: src/main/java/com/mycompany/app/App.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2019 Esri
+ * Copyright 2021 Esri
```
Comment (Rachael-E) [inline src/main/java/com/mycompany/app/App.java:2]: we don't usually update copyright for changes to an existing source code file: that's the guidance I've had in the past from both samples and doc team. We should only change this if it's brand new code.

File: src/main/java/com/mycompany/app/App.java [hunk 3]
```diff
@@ -16,6 +16,8 @@
 
 package com.mycompany.app;
 
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
```
Comment (Rachael-E) [inline src/main/java/com/mycompany/app/App.java:20]: these are fine but there's an unused import below now we're using BasemapStyle

import com.esri.arcgisruntime.mapping.Basemap;

File: src/main/java/com/mycompany/app/App.java [hunk 4]
```diff
@@ -48,12 +50,18 @@ public void start(Stage stage) {
         Scene scene = new Scene(stackPane);
         stage.setScene(scene);
 
+        // An API key is required to enable access to services, web maps, and web scenes hosted in ArcGIS Online.
```
Comment (Rachael-E) [inline src/main/java/com/mycompany/app/App.java:53]: I think we still need that standard line in here about it not being good practice to store the API key in source code.

File: src/main/java/com/mycompany/app/App.java [hunk 5]
```diff
@@ -48,12 +50,18 @@ public void start(Stage stage) {
...
+        String yourApiKey = "YOUR_API_KEY";
+        ArcGISRuntimeEnvironment.setApiKey(yourApiKey);
+
         // create a MapView to display the map and add it to the stack pane
         mapView = new MapView();
         stackPane.getChildren().add(mapView);
 
-        // create an ArcGISMap with the default imagery basemap
-        ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+        // create an ArcGISMap with a basemap
```
Comment (Rachael-E) [inline src/main/java/com/mycompany/app/App.java:63]:
```suggestion
        // create an ArcGISMap with an imagery basemap
```

**Review by Rachael-E — COMMENTED** [review]
a few suggestions @mbcoder thanks!

**Review by Rachael-E — APPROVED** [review]
@mbcoder looks good!

**Review by colinanderson — COMMENTED** [review]
The plugin at https://github.com/Esri/java-gradle-starter-project/pull/14/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R19 is out of date. There is a 0.0.9 version now.

**Comment by mbcoder** [comment]
@colinanderson good catch, plugin updated.

**Review by colinanderson — APPROVED** [review]
@mbcoder looks ok.

**Pull request merged**